### ISSUE TITLE
Contextual information

### ISSUE THREAD

**Issue by nebogeo** [nebogeo/seeme#17]
We need to describe why we are asking people to play the game, perhaps something about the theories of camouflage and what we hope to discover from the results. 

Written for a wide audience, not too wordy and lots of pictures - we can help with editing this if you want, but better if it comes from you originally.

**Comment by scienceanna** [comment]
There's now a basic draft of this in the shared folder - let us know what you think!

**Comment by nebogeo** [comment]
Thanks! I've put the first section in and some pictures - I reckon the rest is a bit too much text for the game but it should be elsewhere (and added a button for a link), do you have a site for your research group? Is it the sensory ecology website?

**Comment by scienceanna** [comment]
On my screen the logos are falling off the bottom of the game, and I can't find the text/pictures anywhere (might be missing something!) We can probably create a simple wordpress site or something for the extra information?

**Comment by nebogeo** [comment]
I've changed the dimensions slightly for the screen size which may help - the info is on the "more info" screen, which also come up when you've played the game...

**Comment by scienceanna** [comment]
Cool, seems to be working now, thanks - think I just needed to clear my cache :)

**Comment by nebogeo** [comment]
Ah great... 
It's not too much work to add this information to the game - it's just that a normal site would be a better medium for this kind of information really, and it would be good if it was associated with your existing work etc.

**Comment by scienceanna** [comment]
Yep, seems sensible to me! Laura and I will work on that and send you a link to add if that's okay.

**Comment by nebogeo** [comment]
Sure thing!
